rpc/Cargo.toml

```diff
@@ -9,6 +9,9 @@ thiserror = "1.0.40"
 jsonrpsee = { version = "0.20", features = ["client-core", "macros"] }
 celestia-types = { workspace = true }
 serde = { version = "1.0.188", features = ["derive"] }
+libp2p = "0.52.3"
+libp2p-identity = "0.2.3"
+bs58 = "0.5.0"
```

Comment:
they really use it? :open_mouth:

Reply:
yeah :D https://docs.libp2p.io/concepts/fundamentals/peers/#how-are-peer-ids-represented-as-strings